packages/components/src/SubHeaderBar/InputTitleSubHeader/InputTitleSubHeader.component.js

```diff
@@ -1,14 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import Skeleton from '../../Skeleton';
 import TitleSubHeader from './TitleSubHeader.component';
 import InlineFormSubHeader from './InlineFormSubHeader.component';
 import theme from './InputTitleSubHeader.scss';
 import Icon from '../../Icon';
 
-function InputTitleSubHeader({ title, iconId, editMode, ...rest }) {
+function InputTitleSubHeader({
+	title,
+	iconId,
+	editMode = false,
+	loading = false,
+	inProgress = false,
+	...rest
+}) {
+	if (loading) {
+		return <Skeleton type={Skeleton.TYPES.text} size={Skeleton.SIZES.large} />;
+	}
+	const subheaderDetailsCn = classNames({
```

Comment:
use rather classNames(theme['tc-subheader-details'], 'tc-subheader-details', { [theme['tc-subheader-details-blink']]: inProgress, 'tc-subheader-details-blink': inProgress}) to use a object only for dynamic parts